CHANGELOG.asciidoc

```diff
@@ -3,6 +3,55 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.17.6]]
+=== Beats version 7.17.6
+https://github.com/elastic/beats/compare/v7.17.5\...v7.17.6[View commits]
+
+==== Breaking changes
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix OS name reported by add_host_metadata on Windows 11. {issue}30833[30833] {pull}32259[32259]
+- Fix race condition when reloading runners. {pull}32309[32309]
+- Fix race condition when stopping runners. {pull}32433[32433]
+
+*Auditbeat*
+
+- auditd module: Fix parsing of audit rules where arguments are quoted (like file paths containing spaces). {pull}32421[32421]
+- auditd module: Fix minimum AuditStatus length so that library can support kernels from 2.6.32. {pull}32421[32421]
+- system/socket: Reduce memory usage of the dataset. {issue}32191[32191] {pull}32192[32192]
+- libbeat/processors/add_process_metadata: Fix memory leak in process cache. {issue}24890[24890] {pull}29717[29717]
+
+*Filebeat*
+
+- Adding a fix for threatintel module where MISP was paginating forever. {pull}31784[31784]
+- Fix handling and mapping of syslog priority, facility and severity values in Cisco module. {pull}32025[32025]
+- Fix http_endpoint input TLS handshake failures. {pull}32105[32105]
+- Fix wrong state ID in states registry for awss3 s3 direct input. {pull}32164[32164]
```

Comment:
I see this twice but want to make sure awss3 is not a typo.

Reply:
awss3 is the input name I believe but I'll reword it to AWS S3 as that will be clearer in the changelog.